File: Back-end/user_service/src/main/java/com/springboot/entities/Services.java
package com.springboot.entities;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Table(name = "Services")
@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor

public class Services {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long service_id;
	private String service_description;
	private float service_price;
	private int service_duration;
	private int category_id;
	private String service_name;

}

File: Front-end/barbershop/src/app/services.ts
export class Services {
     service_id:number;
	 service_description:string;
	 service_price:number;
	 service_duration:number;
	 category_id:number;
	 service_name:string;
     constructor() {
        this.category_id=0;
        this.service_id=0;
        this.service_description="";
        this.service_duration=0;
        this.service_name="";
        this.service_price=0;
     }
}
